src/Resources/translations/VerifyEmailBundle.pl.xlf

```diff
@@ -4,19 +4,19 @@
         <body>
             <trans-unit id="1">
                 <source>%count% year|%count% years</source>
-                <target>%count% roku|%count% lat</target>
+                <target>%count% roku|%count% lata|%count% lat</target>
             </trans-unit>
             <trans-unit id="2">
                 <source>%count% month|%count% months</source>
-                <target>%count% miesiąca|%count% miesięcy</target>
+                <target>%count% miesiąc|%count% miesiące||%count% miesięcy</target>
             </trans-unit>
             <trans-unit id="3">
                 <source>%count% day|%count% days</source>
-                <target>%count% dnia|%count% dni</target>
+                <target>%count% dzień|%count% dni</target>
```

Comment:
you still have only 2 variants here, which is broken if Symfony decides it needs the third one

Reply:
As wrote here https://github.com/SymfonyCasts/verify-email-bundle/pull/56#discussion_r577870396 only two forms are required. But if you need 3 forms to work properly, then the second and third would be the same in this context.

Reply:
thanks guys!